xbmc/cores/AudioEngine/AESinkFactory.h

```diff
@@ -42,15 +42,27 @@ struct AESinkRegEntry
   Cleanup cleanupFunc;
 };
 
+struct AESinkDevice
+{
+  std::string driver;
```

Comment:
What about making these private and adding public setters and getters?

Reply:
I think it could be part of a followup refactor PR to extensively replace the all-in-one device strings (similar to one of your commits).
There isn't much meat in AESinkDevice at this time and it's not the focus of the PR.

Reply:
I agree with @CrystalP . We should not mixup refactors and functionality. This will only make it very hard to backport / cherry-pick parts of it. You can see that in the EAC3 backport scenario ... while the change would have been 8 lines, it was then  > 50. All good, if nothing happens, but not so much if a bug slips in.